layout/js/map.js

```diff
@@ -50,10 +47,28 @@ var animate_bars = function(data) {
     var bar = $("#bar-" + myYear);
     if (bar !== undefined) {
       bar.css(style);
+      if (myYear !== year) {
+        bar.css('opacity','0.25');
+      } else {
+        bar.css('opacity','1.0');
+      }
     }
   });
 };
 
+
+var setStateValue = function(state) {
+  $.get( "js/scaleFactors.json", function( data ) {
```

Comment:
BTW, I haven't merged this, so it is just local right now (so I didn't crash anything). But I could use some help refactoring this so it makes sense.